### DISCUSSION TITLE
Video Downloadhelper does not recognize age-restricted videos on YouTube

### DISCUSSION THREAD

**Discussion by at3671** [aclap-dev/video-downloadhelper#1242] [Help and Q&A]
### Your question

If a YouTube video is labelled "Age-restricted video (based on Community Guidelines)", Video Downloadhelper does not even show that the video exists. 

"No media to process in the current tab - Click play on video to help detect files…" shows when you click on the Video Downloadhelper icon in the upper right hand corner of the YouTube web page loaded in Firefox.

For example, I'll provide links to two videos by the rock band Therion. The first is not age restricted. Video Downloadhelper will recognize that it's available for download. The second video is age restricted. Video Downloadhelper won't even show that it exists.

Hopefully, there's a solution to this problem. Here are the example videos:

https://www.youtube.com/watch?v=QFD7YIk_9dw - not age restricted and comes up in Video Downloadhelper
https://www.youtube.com/watch?v=scdffAiRllY - age restricted and does not come up in Video Downloadhelper

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"4805cf03cb374065"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\eyero\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 17,
    "successfull_dl": 36
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You are reporting a problem that has been reported many times before.  Search this forum with `"age restricted"` as your search key.  The quotation marks are part of the search key.  If you're really ambitious, you could search our old forum on Google Groups for older reports of the same problem.  It was first reported rather a long time ago.

**Reply by at3671** [reply]
It's a shame that there's a problem with no solution, with Video Downloadhelper.
